app/konnect/mesh-manager/migrate-zone.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Migrate a self-managed zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+If you already have zone control planes in {{site.mesh_product_name}}, you can migrate them to {{site.konnect_short_name}} in Mesh Manager. [Mesh Manager](/konnect/mesh-manager/) allows you to create, manage, and view your {{site.mesh_product_name}} service meshes using the {{site.konnect_short_name}} platform.
+
+Here are a few benefits of creating a mesh deployment in {{site.konnect_short_name}} instead of using a self-managed setup:
+
+* **Kong-managed global control plane:** By creating your mesh in {{site.konnect_short_name}}, your global control plane is managed by Kong. 
+* **All entities in one place:** You can view all your information, such as entities from {{site.kic_product_name}} (KIC) for Kubernetes, {{site.konnect_short_name}}-managed entities, and now service mesh data, all from one central platform. 
+* **Managed UI wizard setup:** {{site.konnect_short_name}} simplifies the process of creating a mesh by providing a setup wizard in the UI that guides you through the configuration steps.
+
+This guide explains how to migrate a self-managed zone control plane to {{site.konnect_saas}}. 
+
+![mesh migration before](/assets/images/diagrams/diagram-mesh-migration-before.png)
+
+> **Figure 1:** Diagram that explains how the {{site.mesh_product_name}} global control plane communicates with zone control planes, services, and data planes before migrating. Each service has an associated data plane proxy and those communicate with the zone control plane. The zone control plane then communicates with the {{site.mesh_product_name}} global control plane. All of these entities are self-managed in {{site.mesh_product_name}}.
+
+![mesh migration after](/assets/images/diagrams/diagram-mesh-migration-after.png)
+
+> **Figure 2:** Diagram that explains how the {{site.konnect_short_name}} global control plane communicates with zone control planes, services, and data planes after migrating. Each service has an associated data plane proxy and those communicate with the zone control plane. The services, data plane proxies, and zone control planes are all self-managed in {{site.mesh_product_name}}. The zone control plane then communicates with the {{site.konnect_short_name}} global control plane, which is managed by Kong.
```

Comment:
@lobkovilya If you wouldn't mind reading these over to make sure these sound accurate, that would be great!

Reply:
I'm not sure the phrase "... are self-managed in Kong Mesh" actually makes sense. "Self-managed" means it's managed by the customer (not by Kong), so I don't think we can say "self-managed in <some_place>". I'd rather drop the clarification and keep only "... are self-managed."

Other than that it's very accurate!